enrolments/validators.py

```diff
@@ -0,0 +1,43 @@
+from datetime import datetime
+from django.core.exceptions import ValidationError
+from registration.models import Student
+
+
+def validate_attendance(classObj):
+    """
+    Validates against the attendance column for the Class model.
+    The expected structure is:
+    [
+        {
+            "date": "2021-04-19",
+            "attendees": [
+                // student IDs
+                1, 2, 3, 4, 5
+            ]
+        },
+        {
+            "date": "2021-04-26",
+            "attendees": [
+                // student IDs
+                1, 2, 3, 4, 5
+            ]
+        }
+    ]
+    """
+    for session in classObj:
+        date = session["date"]
+        attendees = session["attendees"]
```

Comment:
the admin throws a `KeyError` if users don't specify either key in their JSON - can we catch this and raise a validation error?

<img width="1536" alt="Screen Shot 2021-03-01 at 1 11 56 AM" src="https://user-images.githubusercontent.com/37346399/109458620-25b91100-7a2b-11eb-9e4c-1b61827442b6.png">